.tests/compatibility/resources.yaml

```diff
@@ -0,0 +1,6 @@
+default_resources:
+  threads: 1
+  time: "12:00:00"
+  mem_mb: 1024
+  mem_per_cpu: 3072
```

Comment:
mem_per_cpu should be the same as mem_mb

Reply:
fixed